src/lib/markbind/src/lib/markdown-it-shared/markdown-it-escape-special-tags.js

```diff
@@ -0,0 +1,95 @@
+const markdownIt = require('../markdown-it');
+
+/*
+ Custom patch for the api to escape content in certain special tags
+ Adapted from the default markdown-it html_block rule and replaces it.
```

Comment:
Apologies, should have really highlighted this.
The specific file is `\node_modules\markdown-it\lib\rules_block\html_block`

75% of this file ( all of the comments ) is essentially the same as it.
The patch here essentially just uses the collected special tags to form a new regex to replace the first one in the original rule.

The htmlparser2 patch has a lot of changes though, 'thankfully' 😅